yellowbrick/cluster/silhouette.py

```diff
@@ -206,4 +214,4 @@ def finalize(self):
 
         # Set the ticks on the axis object.
         self.ax.set_yticks([])  # Clear the yaxis labels / ticks
-        self.ax.set_xticks(np.linspace(-1,1,11))
+        self.ax.xaxis.set_major_locator(plt.MultipleLocator(0.1))  # Set the ticks at multiples of 0.1
```

Comment:
The `MultipleLocator` is a great strategy to use here! The one thing I would ask is that we modify the ticker on the subplot axes (`self.ax`) instead of the whole `plt`, since we want to leave things flexible for plots composed of multiple subplots containing multiple visualizers. It looks like you can achieve this using the strategy shown [here](https://matplotlib.org/gallery/ticks_and_spines/tick-locators.html), by adding `import matplotlib.ticker as ticker`  to the import statements at the top and changing this line to `self.ax.xaxis.set_major_locator(ticker.MultipleLocator(0.1))`.

Reply:
I completely agree with this strategy. I kind of missed the scenario where there can be multiple subplots. Doing this right away.